client/new-stack-editor/lib/index.coffee

```diff
@@ -25,12 +33,19 @@ module.exports = class StackEditorAppController extends AppController
 
     super options, data
 
+    @templates = {}
```

Comment:
This was a temporary thing to show stack template contents with-in this PR, I'm planning to make a controller which uses a generic storage (it can use `reactor` singleton if it's already shaped up at there) safely ignore this part. I've some stuff around here as just "experimental"